packages/media/src/webrtc/Session.ts

```diff
@@ -297,23 +298,24 @@ export default class Session {
         const senders = pc.getSenders();
         // If we didn't specify oldTrack, replace with first of its kind
         if (!oldTrack) {
-            oldTrack = (senders.find((s: any) => s.track && s.track.kind === newTrack.kind) || {}).track;
+            oldTrack = (senders.find((s: RTCRtpSender) => s.track && s.track.kind === newTrack.kind) || {}).track;
```

Comment:
would be cool to type `this.pc` instead, but it looks like that introduces a bunch of type errors because of it's nullability :(

Reply:
We should, but maybe get this fix out first?